.golangci.yaml

```diff
@@ -15,6 +15,8 @@
 linters:
   disable:
     - errcheck # Currently too many places are ignoring errors
+  enable:
+    - nolintlint
```

Comment:
By enabling this linter `golangci-lint` will warn if any `nolint` comments become stale and are no longer required.

It looks like it is complaining about some of the existing `nolint` comments.

Reply:
Updating `golangci-lint` to the latest patch version has resolved the problem.

Reply:
Very nice! And thanks for explaining!